R/climate.R

```diff
@@ -0,0 +1,108 @@
+#' Function for getting climate data from one place or state
+#'
+#' This function gets unofficial daily records from U.S. National Weather Service COOP stations. Note this is unofficial data.
+#'
+#'#' @param place COOP station ID. You can also look up by name of a city or town station nameby setting the is_station_id argument to FALSE. Leave as default "" to return entire state's results. See https://mesonet.agron.iastate.edu/COOP/extremes.php for available states and stations
```

Comment:
> is there a way to find COOP station ID by city?

Not that I'm aware of. And that's why I Web-scraped with htmltab instead of using the JSON Web service. 1) With the Web service, you have to know the station ID, and I don't know how you'd find out the station ID except to download a state's worth of data and then find it. 2) I like being able to search by place name or at least have the place name be returned in results. _The PHP Web page includes the place name in results. JSON Web service doesn't._ And I couldn't find a lookup table. So the only thing I could think to do is create my own lookup table somehow with existing data - possible, but then it becomes incomplete if a new station is added. How would you recommend handling this? Or do you know of a lookup table I could use?

Reply:
Maybe we can ask @akrherz? @akrherz, is there a way to look up COOP stations ids from a JSON webservice?

Reply:
@maelle I don't have any type of search JSON feature, you can only get the listing of sites via per-state JSON service like so: https://mesonet.agron.iastate.edu/geojson/network/IA_COOP.geojson to get Iowa COOP sites.